wagtail/contrib/frontend_cache/backends.py

```diff
@@ -100,7 +100,7 @@ def purge_batch(self, urls):
 
         except requests.exceptions.HTTPError as e:
             for url in urls:
-                logger.error("Couldn't purge '%s' from Cloudflare. HTTPError: %d %s", url, e.response.status_code, e.message)
+                logger.error("Couldn't purge '%s' from Cloudflare. HTTPError: %d %s", url, e.response.status_code, str(e.args[0]))
```

Comment:
Does it make sense to replace it with something like `logging.exception("Couldn't purge '%s' from Cloudflare. HTTPError: %d", url, e.response.status_code)`?

It should log the traceback from the exception as well.

Reply:
Yea, I just saw that the message is contained in `args[0]` so I thought I would add it in. But we can just log the `status_code` if that makes more sense

Reply:
@kevinhowbrook  My point was mostly about using `logging.exception` instead of `logger.error`. The message will be logged using the same logging level (error), but exception info will also added to the logging message.

Reply:
Ah my bad, I've switched that to your rec